List/LocationList.py

```diff
@@ -275,45 +275,45 @@ def getTreasureList():
             KH2Treasure(489, "Crystal Fissure The Great Maw Map", LocationTypes=[locationType.HB]),
             KH2Treasure(180, "Crystal Fissure Energy Crystal", LocationTypes=[locationType.HB]),
             KH2Treasure(181, "Crystal Fissure AP Boost", LocationTypes=[locationType.HB]),
-            KH2Treasure(525, "1000 Heartless Secret Ansem Report 1", LocationTypes=[locationType.HB], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(269, "1000 Heartless Ice Cream", LocationTypes=[locationType.HB], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(511, "1000 Heartless Picture", LocationTypes=[locationType.HB], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(491, "Postern Gull Wing", LocationTypes=[locationType.HB]),
+            KH2Treasure(525, "1000 Heartless Secret Ansem Report 1", LocationTypes=[locationType.HB], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(269, "1000 Heartless Ice Cream", LocationTypes=[locationType.HB], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(511, "1000 Heartless Picture", LocationTypes=[locationType.HB], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(491, "Postern Gull Wing", LocationTypes=[locationType.HB]).setLocationWeight(10),
             KH2Treasure(311, "Heartless Manufactory Cosmic Chain", LocationTypes=[locationType.HB], InvalidChecks=[itemType.MEMBERSHIPCARD]),
```

Comment:
Shouldn't this have late game weight?

Reply:
Since this is a hybrid mid/late check, I just biased toward making it a mid-game check.